src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -181,15 +183,15 @@ rpc::PlacementGroupStats *GcsPlacementGroup::GetMutableStats() {
 
 GcsPlacementGroupManager::GcsPlacementGroupManager(
     instrumented_io_context &io_context,
-    std::shared_ptr<GcsPlacementGroupSchedulerInterface> scheduler,
+    GcsPlacementGroupSchedulerInterface *scheduler,
```

Comment:
This is never nullptr, so use`&`, not `*` ?

Reply:
Same here, we have places passing in nullptr.